### DISCUSSION TITLE
Modify test tolerances

### DISCUSSION THREAD

**Discussion by aarograh** [idaholab/moose#21949] [Q&A Tools] [answered]
Hello, I'm having some challenges setting some test tolerances.  I've tried `difference_tol` and `ratio_tol`, both of which are in the output of `run_tests --dump`, and both of them result in this:
```
tests:32: unused parameter "difference_tol"
tests:33: unused parameter "ratio_tol"
```
Conversely, I tried setting `abs_zero` to a very high value since it appears in the dump just like the other 2.  This parameter worked and caused the test to pass.

Any ideas why those parameters aren't working?  I'm sure I'm misunderstanding something but this is the first time I'm even remotely delving into the test harness.

**Comment by GiudGiud** [comment, marked as answer]
Hello

there’s a difference between the parameter names in the test harness and in the utility the test harness leverages. 

abs_zero and rel_~~tol~~err can be parsed by the test harness. I haven’t seen the other two. If you have seen them elsewhere in the repository could you please point me to them?

Guillaume

**Reply by GiudGiud** [reply]
It’s weird they are in the dump though. I ll check when I get back

**Reply by aarograh** [reply]
Thanks for the information, I'll give those names a try.  It would definitely be helpful if the `--dump` names reflected the actual names that could be used in the file.

**Reply by aarograh** [reply]
@GiudGiud I tried `rel_tol` and still got `unused parameter "rel_tol"` in the test output so it seems like that didn't work.

**Reply by GiudGiud** [reply]
so ratio_tol and difference_tol are only for Jacobian exactness testing. 

my bad, it's not rel_tol but rel_err

**Reply by aarograh** [reply]
Oh, got it.  `rel_err` is listed in the dump as well, so that's good.  Thank you!

**Reply by GiudGiud** [reply]
Yeah I double checked the dump. 
It's organized by test type, the parameters you used previously are under the test type AnalyzeJacobian only
